### PR TITLE
Adds CBD

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Credit to Goon for this.

Adds cbd to cannabis--it's a non-psychoactive drug that helps you relax and also promotes healing; it also has mild anti-stun properties to it.

Also adds a contented sigh emote

:cl: Fox McCloud
add: Marijuana plants now have cbd in them; it's a non-psychoactive/legal drug that promotes relaxation and healing
add: Adds a contented sigh emote *hsigh
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -585,6 +585,14 @@
 					message = "<B>[src]</B> makes a weak noise"
 					m_type = 2
 
+		if("hsigh", "hsighs")
+			if(!muzzled)
+				message = "<B>[src]</B> sighs contentedly."
+				m_type = 2
+			else
+				message = "<B>[src]</B> makes a weak noise"
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/emote.dm]: `makes a chill/relaxed noise`

**Comment by KasparoVy** [comment]
Blaze it

**Comment by Mitchs98** [comment]
Dave's not here man.

File: code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (Fox-McCloud) [reply code/modules/mob/living/carbon/human/emote.dm]: good suggestion!

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -590,7 +590,7 @@
 				message = "<B>[src]</B> sighs contentedly."
 				m_type = 2
 			else
-				message = "<B>[src]</B> makes a weak noise"
+				message = "<B>[src]</B> makes a [pick("chill", relaxed")] noise"
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/emote.dm]: Hell yeah!

**Comment by pinatacolada** [comment]
Should also make you hungry

**Comment by Carthusia** [comment]
heh, you get a little moodle in the top right under your hunger bar that notes you got the "Munchies"

To confirm though: This will remove the spacedrug like effect from it and replace it with the proper CBD effect?

**Comment by Fox-McCloud** [comment]
THC is still around; its still does stereotypical psycho-active THC things.

This is a completley and utterly separate chem.

File: code/modules/reagents/chemistry/reagents/drugs.dm [hunk 3]
```diff
@@ -517,6 +517,32 @@
...
+	if(prob(5))
+		to_chat(M, "<span class='notice'>[pick("You feel peaceful.", "You breathe softly.", "You feel chill.", "You vibe.")]</span>")
+	if(prob(10))
+		M.AdjustConfused(-5)
+		update_flags |= M.SetWeakened(0, FALSE)
+	if(volume >= 70 && prob(25))
+		if(M.reagents.has_reagent("thc") <= 20)
+			M.Drowsy(10)
+	if(prob(25))
+		update_flags |= M.adjustBruteLoss(-2, FALSE)
```
Comment (Kyep) [inline code/modules/reagents/chemistry/reagents/drugs.dm]: This healing seems pretty substantial. Even Saline Glucose, which is the basic medical chem, heals only 0.66 brute/burn per tick. This heals 0.5 brute+burn per tick, as well as reducing confusion. Not sure about its effects on weakening, too. 

Is this meant to effectively replace most brute/burn treatment for botanists, and whoever they give it to?

**Comment by SynthNumerics** [comment]
for clarification on cbd itself, its actually the chemical that prevents/limits the high that you gain from thc- the chemical you typically think of when you consider cannabis that makes you super out of it, get peckish etc; a certain level of it is honestly needed to prevent "bad trips". cbd gives a kind of "body high" where your mind is clear but your body as mentioned gains a level of relaxation with its pretty potent painkiller an mild healin properties (i use it daily to help manage my pain) among other things that aid the health of the body (i can talk more abt it on disc if folks want). it's a bit of a miracle chemical in a way.

im feelin a lil late but regardin how it affects the game however, i have questions.
1. would the cbd be present in premade (ie medicinal from the psych office) cigarettes an would they be altered for the straight cbd version due to its connection or would they retain both chemicals in certain percentages for both effects?
2. does it have the blockin affect on thc or does it remain independent in this case?

**Pull request merged**